int32 device_address
---
bool[] status